src/libs/ajax.js

```diff
@@ -1110,9 +1110,23 @@ const DataRepo = signal => ({
       details: async () => {
         const res = await fetchDataRepo(`repository/v1/snapshots/${snapshotId}`, _.merge(authOpts(), { signal }))
         return res.json()
+      },
+      exportSnapshot: async () => {
+        const res = await fetchDataRepo(`repository/v1/snapshots/${snapshotId}/export`, _.merge(authOpts(), { signal }))
+        return res.json()
       }
     }
-  }
+  },
+  job: jobId => ({
+    details: async () => {
+      const res = await fetchDataRepo(`repository/v1/jobs/${jobId}`, _.merge(authOpts(), { signal }))
+      return res.json()
+    },
+    result: async () => {
+      const res = await fetchDataRepo(`repository/v1/jobs/${jobId}/result`, _.merge(authOpts(), { signal }))
```

Comment:
Same question here: probably need some error handling?

Reply:
Added! (We do it at call site, not in the ajax library itself, usually)